examples/fabric/v4/cloudRouterConnectivity/cloudRouter2serviceprofile/main.tf

```diff
@@ -0,0 +1,50 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+data "equinix_fabric_service_profiles" "sp" {
+  filter {
+    property = "/name"
+    operator = "="
+    values = [var.fabric_sp_name]
```

Comment:
Nit: the examples would benefit from running `terraform fmt` to ensure they line up with the formatting standards recommended by HashiCorp.